src/unix/bsd/freebsdlike/freebsd/mod.rs

```diff
@@ -1351,6 +1340,32 @@ s! {
 }
 
 s_no_extra_traits! {
+    #[cfg_attr(feature = "extra_traits", derive(Debug))]
+    pub struct aiocb {
+        pub aio_fildes: ::c_int,
+        pub aio_offset: ::off_t,
+        pub aio_buf: *mut ::c_void,
+        pub aio_nbytes: ::size_t,
+        __unused1: [::c_int; 2],
+        __unused2: *mut ::c_void,
+        pub aio_lio_opcode: ::c_int,
+        pub aio_reqprio: ::c_int,
+        // unused 3 through 5 are the __aiocb_private structure
+        __unused3: ::c_long,
+        __unused4: ::c_long,
+        __unused5: *mut ::c_void,
+        pub aio_sigevent: sigevent,
+    }
+
+    // Can't correctly impl Debug for unions
+    #[allow(missing_debug_implementations)]
+    pub union __c_anonymous_sigev_un {
```

Comment:
```suggestion
    pub union __c_anonymous_sigev_union {
```

Might as well be consistent with the other instance which spells out `union` in the type name but leaves calls the field `_un`

Reply:
Nevermind, makes more sense to keep the field name like we do for anonymous structs. I guess the other one should change.